doc/openapi-user-guide.md

```diff
@@ -147,23 +147,25 @@ spec:
       serviceAccountsEnabled: true
       directAccessGrantsEnabled: true
 ```
-- **oidc** is optional field in OpenAPI CR, Only for OIDC.  
-- **issuerEndpointRef** - Secret, that contains **issuerEndpoint**
-
+- **oidc** is optional field in OpenAPI CR
+- Only for OIDC: 
 
 | **Field**                | **Required** | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                       |
-| --- | --- | --- |
+| --- | --- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Extra hiphens added here . I know this was rasised before and the IDE is doing it. You can disable table formatting in the ide settings to stop this from happening
![image](https://github.com/3scale/3scale-operator/assets/16667688/8dddb52b-4040-42bd-a8ed-3410a6f00ea7)

Reply:
My bad after testing it doesn't work :facepalm:

Reply:
Updated. Thank you for review and help notes.